src/core/PWSprefs.cpp

```diff
@@ -132,6 +132,8 @@ const PWSprefs::boolPref PWSprefs::m_bool_prefs[NumBoolPrefs] = {
   {_T("ListSortAscending"), true, ptApplication},           // application
   {_T("EnableWindowTransparency"), false, ptApplication},   // application
   {_T("ShowMenuSeparator"), true, ptApplication},           // application
+  {_T("OptimizedCellSize"), false, ptApplication},          // application
```

Comment:
Why is this a user-visible preference? When would a user set or clear this?

Reply:
This is to set at users demand before using the grid view or while he is working the first time with it. Either the screen is big enough to expand and scrolling in the grid view is not needed or no problem for the user or the user is not using this key and align the column wide by his own. A command in the menu might be also an idea, instead of hiding in the options menu.